cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Linear solver tolerance                = 1e-5
+set Max nonlinear iterations               = 10 
+set Temperature solver tolerance           = 1e-4
+set Number of cheap Stokes solver steps    = 0
+
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+
+subsection Geometry model
+  set Model name = ellipsoidal chunk
+  
+  subsection Ellipsoidal chunk
+   set  NE corner = 131:-28
+   set  NW corner = 109:-28
+   set  SW corner = 109:-41
+   set  SE corner =  
+  end
+end   
+
+   
+subsection Initial temperature model
+  set Model name = adiabatic boundary
+  subsection Adiabatic boundary
+    set Isotherm depth filename = adiabatic_boundary.txt
```

Comment:
This file should be included in the pull request. As I understand it from our discussion, the file was derived from the LITH1.0 data set. Would you also be willing to include the script used to generate the .txt file in this pull request?